oximeter/db/src/lib.rs

```diff
@@ -335,11 +335,13 @@ pub(crate) fn timeseries_key_for<'a>(
     target_fields: impl Iterator<Item = &'a Field>,
     metric_fields: impl Iterator<Item = &'a Field>,
 ) -> TimeseriesKey {
-    use std::collections::hash_map::DefaultHasher;
+    use highway::HighwayHasher;
     use std::hash::{Hash, Hasher};
-    let mut hasher = DefaultHasher::new();
+    let mut hasher = HighwayHasher::default();
     for field in target_fields.chain(metric_fields) {
-        field.hash(&mut hasher);
+        bcs::to_bytes(&field)
+            .expect("Failed to serialized field to bytes")
```

Comment:
This `expect` ideally would be cleaned up, but I was a bit lazy because of this comment:

> Serialization can fail if T’s implementation of Serialize decides to fail, if T contains sequences which are longer than MAX_SEQUENCE_LENGTH, or if T attempts to serialize an unsupported datatype such as a f32, f64, or char.

This seems unlikely to me because `MAX_SEQUENCE_LENGTH` is `2_147_483_647usize`, and we try to exhaustively test enum variants down below. But if y'all would prefer, I can plumb the error propagation back up.

Reply:
Expect is fine to me.